modes/longitudinal/src/toolbarButtons.js

```diff
@@ -326,12 +316,23 @@ const toolbarButtons = [
       type: 'tool',
       icon: 'tool-crosshair',
       label: 'Crosshairs',
+      // TODO - figure out what it takes here to deactivate crosshairs
+      volumeDeactivate: {
+        interactionType: 'tool',
+        commands: [
+          {
+            commandOptions: {
+              toolName: 'WindowLevel',
+            },
+            context: 'CORNERSTONE',
+          },
+        ],
+      },
```

Comment:
I rather have a `callback` that can specify a `event` that it can subscribe to and run a command than hardcoing the `volumeDeactiate`, the general pattern of doing something on an event is pretty usefull, not like this though

Reply:
What about a set of commands to run when the HP changes?  That belongs here, and means one doesn't need an extra listener that specifically knows about crosshairs.   In fact, what has been done here is to add a very specific listener.

Reply:
Regardless, it is a hack until you get your GUI changes to deal with the menu changing based on the hanging protocol - once that is done, it should work fine without any customization at all.

Reply:
I guess I like the `hangingProtocolChangeCallbacks: []` much better than `volumeDeactivate`